talend-component-maven-plugin/src/main/java/org/talend/sdk/component/maven/IconReporterMojo.java

```diff
@@ -98,7 +98,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                     if (missingIcon == null) {
                         final ClassLoader fallbackLoader = IconReporterMojo.class.getClassLoader();
                         try (final InputStream stream = fallbackLoader.getResourceAsStream("icon/missing.png")) {
-                            missingIcon = toDataUri(new IconResolver.Icon("image/png", IO.readBytes(stream)));
+                            missingIcon = toDataUri(new IconResolver.Icon("image/png", IO.readBytes(stream), "light"));
```

Comment:
and here we are using "light" instead of ""

Reply:
The icon reporter mojo is not more in use now. Just the default theme fallback.